2015-05-08 11:27:09,463 INFO: microblog startup [in /home/pattisdr/microblog/app/__init__.py:26]
2015-05-08 11:27:24,664 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    posts=posts)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:27:27,214 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    posts=posts)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:27:30,679 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    posts=posts)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:27:39,904 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    posts=posts)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:27:41,745 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    posts=posts)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:27:44,885 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    posts=posts)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:27:46,855 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    posts=posts)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:28:08,750 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    posts=posts)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:28:18,851 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    posts=posts)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:28:21,587 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    posts=posts)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:28:24,134 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    posts=posts)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:28:25,314 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    posts=posts)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:28:30,565 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    posts=posts)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:28:36,494 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    posts=posts)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:30:03,075 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    if g.user is not None and g.user.is_authenticated():
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:30:17,186 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    if g.user is not None and g.user.is_authenticated():
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:30:19,081 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    if g.user is not None and g.user.is_authenticated():
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:30:22,551 ERROR: Exception on /index [GET] [in /home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask_login.py", line 758, in decorated_view
    return func(*args, **kwargs)
  File "/home/pattisdr/microblog/app/views.py", line 40, in index
    if g.user is not None and g.user.is_authenticated():
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pattisdr/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/pattisdr/microblog/app/templates/base.html", line 28, in top-level template code
    {% block content %}{% endblock %}
  File "/home/pattisdr/microblog/app/templates/index.html", line 6, in block "content"
    {{ form.hidden_tag() }}
  File "/home/pattisdr/microblog/flask/local/lib/python2.7/site-packages/jinja2/environment.py", line 397, in getattr
    return getattr(obj, attribute)
UndefinedError: 'form' is undefined
2015-05-08 11:32:17,802 INFO: microblog startup [in /home/pattisdr/microblog/app/__init__.py:26]
2015-05-08 11:32:18,496 INFO: microblog startup [in /home/pattisdr/microblog/app/__init__.py:26]
2015-05-08 11:34:29,544 INFO: microblog startup [in /home/pattisdr/microblog/app/__init__.py:26]
2015-05-08 11:35:22,243 INFO: microblog startup [in /home/pattisdr/microblog/app/__init__.py:26]
2015-05-08 11:38:58,700 INFO: microblog startup [in /home/pattisdr/microblog/app/__init__.py:26]
2015-05-08 11:39:32,037 INFO: microblog startup [in /home/pattisdr/microblog/app/__init__.py:26]
2015-05-08 11:40:13,500 INFO: microblog startup [in /home/pattisdr/microblog/app/__init__.py:26]
2015-05-08 11:40:54,005 INFO: microblog startup [in /home/pattisdr/microblog/app/__init__.py:26]
2015-05-08 11:44:41,524 INFO: microblog startup [in /home/pattisdr/microblog/app/__init__.py:26]
2015-05-08 11:44:42,204 INFO: microblog startup [in /home/pattisdr/microblog/app/__init__.py:26]
2015-05-08 11:47:07,448 INFO: microblog startup [in /home/pattisdr/microblog/app/__init__.py:26]
2015-05-08 11:49:35,680 INFO: microblog startup [in /home/pattisdr/microblog/app/__init__.py:26]
2015-05-08 11:53:16,046 INFO: microblog startup [in /home/pattisdr/microblog/app/__init__.py:26]
2015-05-08 11:54:36,136 INFO: microblog startup [in app/__init__.py:26]
